### PR TITLE
LPS-197241 Fix exception thrown in log by deleting the created object definition

### PR DESCRIPTION
**Pull request by devpedrotavares**
Related issue: [LPS-194831](https://liferay.atlassian.net/browse/LPS-194831)

### CODE REVIEW

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java [hunk 1]
```diff
@@ -2075,6 +2075,9 @@ public void testGetObjectEntryRelatedObjectEntriesWithAccountEntryRestricted()
 
 		_objectRelationshipLocalService.deleteObjectRelationship(
 			objectRelationship2.getObjectRelationshipId());
+
+		objectDefinitionLocalService.deleteObjectDefinition(
+			childObjectDefinition);
```
Comment (murilostodolni) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:2080]: In this case, we need only delete the `childObjectDefinition` and the `objectRelationship1` and `objectRelationship2` will be delete in the deleting objectDefinition process.

**Review by murilostodolni — CHANGES_REQUESTED** [review]
Hey @devpedrotavares 

I finished my review. Let me know if you have questions.

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java (hunk 1 above)
Comment (devpedrotavares) [reply modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java]: You're right. Just updated the PR

**Comment by devpedrotavares** [comment]
ci:test:sf

**Review by murilostodolni — APPROVED** [review]
LGTM! Thank you @devpedrotavares

**Review by carolmariaabb — APPROVED** [review]

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by murilostodolni** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**